proof/invariant-abstract/DetSchedSchedule_AI.thy

```diff
@@ -26550,6 +27301,7 @@ lemma call_kernel_valid_sched:
         \<and> cur_sc_offset_sufficient (consumed_time s) s\<rbrace>
    call_kernel e
    \<lbrace>\<lambda>_. valid_sched :: det_state \<Rightarrow> _\<rbrace>"
+  supply comp_apply[simp del]
```

Comment:
I would like to take this opportunity to express concern at the size of DetSchedSchedule_AI on rt. My browser took longer than my tea does to brew in order to open this file's diff for review.